frontend/src/app/utils/testResultCSV.ts

```diff
@@ -79,5 +81,19 @@ export function parseDataForCSV(data: any[]) {
         r.patient.residentCongregateSetting,
       "Patient is employed in healthcare": r.patient.employedInHealthcare,
     };
+
+    return includeRSV
+      ? {
+          ...csvData1,
+          "RSV result":
+            TEST_RESULT_DESCRIPTIONS[
+              getResultByDiseaseName(r.results, "RSV") as Results
+            ],
+          ...csvData2,
+        }
+      : {
+          ...csvData1,
+          ...csvData2,
+        };
```

Comment:
I wanted the RSV result column to come after the Covid, flu A, and flu B columns

called out clean up for this method in #7056